### ISSUE TITLE
Material Maps are broken on Mario Kart Wii

### ISSUE THREAD

**Issue by HenrikoMagnifico** [Tinob/Ishiiruka#111]
After a lot of testing, I've come to the conclusion that Material Maps don't work on Mario Kart Wii at all (which is a shame considering I've made nearly 80 brand new material maps for my new texture pack). 

Ishiiruka-Dolphin 1025
Mario Kart Wii (NTSC)
Tried multiple different graphic backends, none of them work.

Yes, I have "Load Custom Material Maps" enabled in the Advanced menu.

Fixing this issue would be much appreciated! <3

**Comment by Tinob** [comment]
if you use tesselation and  enable forced ligthing it changes this issue?

**Comment by HenrikoMagnifico** [comment]
Nope. The textures work just fine but the material maps won't display. I tried auto-bumps but that didn't work either.
